listenbrainz/labs_api/labs/api/recording_from_recording_mbid.py

```diff
@@ -54,45 +56,84 @@ def fetch(self, params, offset=-1, count=-1):
                     r = dict(row)
                     redirect_index[str(r['recording_mbid_old'])] = str(r['recording_mbid_new'])
 
+                # Now start looking up actual recordings
                 for i, mbid in enumerate(mbids):
                     if mbid in redirect_index:
                         mbids[i] = redirect_index[mbid]
 
                 query = '''SELECT r.gid AS recording_mbid, r.name AS recording_name, r.length, r.comment,
                                   ac.id AS artist_credit_id, ac.name AS artist_credit_name,
-                                  array_agg(a.gid) AS artist_credit_mbids
+                                  array_agg(a.gid) AS artist_credit_mbids,
```

Comment:
you said that you were going to unique these, did you fix the issue of the duplicates?

Reply:
Yes, rather than mess with array_agg(), ugh, I opted for list(set(...)) below